.github/workflows/pulumi-cli.yml

```diff
@@ -0,0 +1,102 @@
+env:
+  GITHUB_TOKEN: ${{ secrets.PULUMI_BOT_TOKEN }}
+
+name: cli docs build
+on:
+  pull_request:
+  repository_dispatch:
+    types:
+      - pulumi-cli
+jobs:
+  pull-request:
+    runs-on: ubuntu-latest
+    needs: build-pulumi-cli-docs
+    steps:
+      - name: checkout docs repo
+        uses: actions/checkout@v2
+      - name: pull-request
+        uses: repo-sync/pull-request@v2
+        with:
+          source_branch: "pulumi/${{ github.run_id }}-${{ github.run_number }}"
+          destination_branch: "master"
+          pr_title: "Regen docs pulumi@${{ github.event.client_payload.ref }}"
+          pr_body: ""
+          pr_label: "automation/pulumi-cli-docs"
+          github_token: ${{ secrets.PULUMI_BOT_TOKEN }}
+
+  build-pulumi-cli-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: set the pulumi version
+        run: |
+          echo "PULUMI_VERSION=2.12.0" >> $GITHUB_ENV
```

Comment:
Is this a specific test??

Reply:
yes this is just for this test, it'll use the payload version usually.